app/views/external_users/claims/buttons/_continue.html.haml

```diff
@@ -1,6 +1,6 @@
 .form-section
   .form-group.form-buttons.button-holder
     - commit = claim.next_step? ? 'commit_continue' : 'commit_submit_claim'
-    = f.submit t('buttons.continue'), id: 'save_continue', name: commit, class: 'button left btn-spacer-20'
+    = govuk_button(id: 'save_continue', name: commit) { t('buttons.continue') }
 
-    = f.submit t('buttons.save_a_draft'), name: 'commit_save_draft', class: 'button-gray-3 left'
+    = govuk_button_secondary() { t('buttons.save_a_draft') }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3605/files#r526000031)